docs/examples/multi_modal/multi_modal_video_RAG.ipynb

```diff
@@ -39,7 +39,8 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    context_str = "".join(txt)
</pre><p>could you keep the outputs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10722/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
yeah I just saw that nb clean removed outputs , adding them back

Reply:
@hatianzhang resolved